kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2006,12 +2006,34 @@ object KyuubiConf {
       .stringConf
       .createWithDefault("file:///tmp/kyuubi/events")
 
+  val SERVER_EVENT_KAFKA_TOPIC: OptionalConfigEntry[String] =
+    buildConf("kyuubi.backend.server.event.kafka.topic")
+      .doc("The topic of server events go for the built-in Kafka logger")
+      .version("1.8.0")
+      .serverOnly
+      .stringConf
+      .createOptional
+
+  val SERVER_EVENT_KAFKA_CLOSE_TIMEOUT: ConfigEntry[Long] =
+    buildConf("kyuubi.backend.server.event.kafka.close.timeout")
+      .doc("Period to wait for Kafka producer of server event handlers to close.")
+      .version("1.8.0")
+      .serverOnly
+      .timeConf
+      .createWithDefault(Duration.ofMinutes(5000).toMillis)
```

Comment:
5000 minutes? what a crazy duration!

Reply:
Oops... Shaking hands on the keyboard. 😂 Fixed the time unit to milliseconds.